pack/rpm.mk

```diff
@@ -129,7 +134,11 @@ package: $(BUILDDIR)/$(RPMSRC)
 	if [ -n "$(PACKAGECLOUD_USER)" ] && [ -n "$(PACKAGECLOUD_REPO)" ]; then \
 		curl -s https://packagecloud.io/install/repositories/$(PACKAGECLOUD_USER)/$(PACKAGECLOUD_REPO)/script.rpm.sh | sudo bash; \
 	fi
-	sudo dnf builddep -y $< || sudo yum-builddep -y $<
+	if [ "$(IS_SUSE)" == "true" ]; then \
+		sudo zypper --non-interactive source-install --build-deps-only --force-resolution --recommends $< ; \
+	else \
+		sudo dnf builddep -y $< || sudo yum-builddep -y $< ; \
+	fi
```

Comment:
Nit: It can be written like `sudo zypper <...> || sudo dnf builddep <...> || sudo yum-builddep <...>` to don't depend on the `IS_SUSE` flang and to be consistent with how dnf / yum tool is choosen here.